docs/components/radios/test.html

```diff
@@ -1,720 +1,578 @@
-            <hx-radio-control>
-              <input
-                type="radio"
-                id="radRequiredEnabledCheckedValidFocus"
-                class="mock-focus"
-                checked
-                required
-              />
-              <label for="radRequiredEnabledCheckedValidFocus">
-                <hx-radio></hx-radio>
-                :required:enabled:checked:valid:focus
-              </label>
-            </hx-radio-control>
-            <hx-radio-control>
-              <input type="radio"
-                id="radRequiredEnabledCheckedValidHover"
-                class="mock-hover"
-                checked
-                required
-              />
-              <label for="radRequiredEnabledCheckedValidHover">
-                <hx-radio></hx-radio>
-                :required:enabled:checked:valid:hover
-              </label>
-            </hx-radio-control>
-          </td>
-          <td>
-            <hx-radio-control>
-              <input
-                type="radio"
-                id="radRequiredEnabledCheckedInvalidNone"
-                checked
-                data-mock-invalid
-                required
-              />
-              <label for="radRequiredEnabledCheckedInvalidNone">
-                <hx-radio></hx-radio>
-                :required:enabled:checked:invalid:(blur)
-              </label>
-            </hx-radio-control>
-            <hx-radio-control>
-              <input
-                type="radio"
-                id="radRequiredEnabledCheckedInvalidFocus"
-                class="mock-focus"
-                checked
-                data-mock-invalid
-                required
-              />
-              <label for="radRequiredEnabledCheckedInvalidFocus">
-                <hx-radio></hx-radio>
-                :required:enabled:checked:invalid:focus
-              </label>
-            </hx-radio-control>
-            <hx-radio-control>
-              <input
-                type="radio"
-                id="radRequiredEnabledCheckedInvalidHover"
-                class="mock-hover"
-                checked
-                data-mock-invalid
-                required
-              />
-              <label for="radRequiredEnabledCheckedInvalidHover">
-                <hx-radio></hx-radio>
-                  :required:enabled:checked:invalid:hover
-              </label>
-            </hx-radio-control>
-          </td>
-        </tr>
-      </tbody>
-    </table>
+    <section>
+      <header>
+        <h2>Optional - Enabled</h2>
+      </header>
+      <table>
+        <thead>
+          <tr>
+            <th>:valid</th>
+            <th>:invalid</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>
+              {{ radioSet('dirty', isDirty=true)}}
+            </td>
+            <td>
+              {{ radioSet('dirty', isDirty=true, isValid=false)}}
+            </td>
+          </tr>
+          <tr>
+            <td>
```

Comment:
reduce indentation, make inline to possibly cut line numbers

Reply:
lines reduced